src/sage/categories/finite_dimensional_lie_algebras_with_basis.py

```diff
@@ -747,6 +747,254 @@ def inner_derivations_basis(self):
             return tuple([matrix(R, N, N, list(b))
                           for b in IDer.row_module().basis()])
 
+        @cached_method
+        def nilradical_basis(self):
+            r"""
+            Return a basis of the nilradical of ``self``.
+
+            .. SEEALSO::
+
+                :meth:`nilradical`
+
+            EXAMPLES::
+
+                sage: scoeffs = {('a','d'): {'a':1}, ('a','e'): {'b':-1},
+                ....:            ('b','d'): {'b':1}, ('b','e'): {'a':1},
+                ....:            ('d','e'): {'c':1}}
+                sage: L.<a,b,c,d,e> = LieAlgebra(QQ, scoeffs)
+                sage: L.nilradical_basis()
+                (a, b, c)
+                sage: L.is_nilpotent()
+                False
+
+                sage: sl3 = LieAlgebra(QQ, cartan_type=['A',2])
+                sage: sl3.nilradical_basis()
+                ()
+
+                sage: scoeffs = {('a','e'): {'a':1}, ('b','e'): {'a':1, 'b':1},
+                ....:            ('c','d'): {'a':1}, ('c','e'): {'c':1}}
+                sage: L.<a,b,c,d,e> = LieAlgebra(QQ, scoeffs)
+                sage: L.nilradical_basis()
+                (a, b, c, d)
+                sage: L.is_solvable()
+                True
+                sage: L.is_nilpotent()
+                False
+
+                sage: K1 = L.quotient([a])
+                sage: K1.nilradical_basis()
+                (b, c, d)
+
+                sage: SL = L.subalgebra([a,b,c,d]); SL
+                Subalgebra generated by (a, b, c, d) of Lie algebra on 5 generators (a, b, c, d, e) over Rational Field
```

Comment:
```suggestion
                Subalgebra generated by (a, b, c, d) of 
                 Lie algebra on 5 generators (a, b, c, d, e) over Rational Field
```

Reply:
throughout I recommend to break long output at column 88 so that it fits in the formatted HTML doc

Reply:
I did this everywhere I felt it was at most a net-zero improvement in readability.